code/jest.config.js

```diff
@@ -2,20 +2,10 @@ const os = require('os');
 
 // TODO Revisit this test later, when we have a windows machine @valentinpalkovic
```

Comment:
The meaning of this comment changed. Can you update it? E.g. Explain what challenge blocks us from fixing the other tests